src/test/java/org/broadinstitute/hellbender/tools/coveragemodel/germline/CoverageModellerGermlineSparkToggleIntegrationTest.java

```diff
@@ -0,0 +1,398 @@
+            "learning_combined_read_counts.tsv");
+    private static final File TEST_LEARNING_SAMPLE_SEX_GENOTYPES_FILE = new File(TEST_SUB_DIR,
+            "learning_sample_sex_genotypes.tsv");
+    private static final File TEST_LEARNING_COMBINED_COPY_NUMBER_FILE = new File(TEST_SUB_DIR,
+            "learning_combined_copy_number.tsv");
+
+    private static final File TEST_CALLING_COMBINED_RAW_READ_COUNTS_FILE = new File(TEST_SUB_DIR,
+            "calling_combined_read_counts.tsv");
+    private static final File TEST_CALLING_SAMPLE_SEX_GENOTYPES_FILE = new File(TEST_SUB_DIR,
+            "calling_sample_sex_genotypes.tsv");
+    private static final File TEST_CALLING_COMBINED_COPY_NUMBER_FILE = new File(TEST_SUB_DIR,
+            "calling_combined_copy_number.tsv");
+
+    private static final File TEST_TRUTH_SIM_MODEL = new File(TEST_SUB_DIR, "sim_model");
+    private static final File TEST_TARGETS_FILE = new File(TEST_TRUTH_SIM_MODEL, "targets.tsv");
+
+    private static final double MAPPING_ERROR_RATE = 5e-4; /* reflects the simulated data */
+    private static final int NUM_LATENTS = 5; /* simulated data uses 3 */
+    private static final int MAX_COPY_NUMBER = 3; /* reflects the simulated data */
+
+    private static final int MIN_LEARNING_READ_COUNT = 10;
+    private static final int MAX_LEARNING_EM_ITERATIONS = 10;
+    private static final int MAX_CALLING_EM_ITERATIONS = 10;
+
+    private static final double MIN_PASS_REF_CONCORDANCE = 0.95;
+    private static final double MIN_PASS_ALT_CONCORDANCE = 0.30;
+    private static final double MIN_PASS_HOM_DEL_CONCORDANCE = 0.95;
+    private static final double MIN_PASS_ABS_CONCORDANCE = 0.30;
+
+    private static final File LEARNING_OUTPUT_PATH = createTempDir("coverage_modeller_germline_learning_output");
+    private static final File CALLING_OUTPUT_PATH = createTempDir("coverage_modeller_germline_calling_output");
+    private static final File LEARNING_MODEL_OUTPUT_PATH = new File(LEARNING_OUTPUT_PATH,
+            CoverageModellerGermlineSparkToggle.FINAL_MODEL_SUBDIR);
+    private static final File LEARNING_POSTERIORS_OUTPUT_PATH = new File(LEARNING_OUTPUT_PATH,
+            CoverageModellerGermlineSparkToggle.FINAL_POSTERIORS_SUBDIR);
+    private static final File CALLING_POSTERIORS_OUTPUT_PATH = new File(CALLING_OUTPUT_PATH,
+            CoverageModellerGermlineSparkToggle.FINAL_POSTERIORS_SUBDIR);
+
+    /* for Spark tests */
+    private static final int SPARK_NUMBER_OF_PARTITIONS = 7;
+    private static final File SPARK_CHECKPOINTING_PATH = createTempDir("coverage_model_spark_checkpoint");
+
+    private static GermlinePloidyAnnotatedTargetCollection GERMLINE_PLOIDY_ANNOTATIONS;
+    private static SexGenotypeDataCollection LEARNING_SEX_GENOTYPES_DATA, CALLING_SEX_GENOTYPES_DATA;
+
+    @BeforeSuite @Override
+    public void setTestVerbosity(){
+        LoggingUtils.setLoggingLevel(Log.LogLevel.INFO);
```

Comment:
Actually, I can set the verbosity to `INFO` as a tool argument; no need to change the global log level.

As a side note -- reverting to the original global log level is currently impossible without some code duplication from `LoggingUtils`. The reason is that `logger.getLevel()` returns `org.apache.logging.log4j.Level` whereas we need `htsjdk.samtools.util.Log.LogLevel` for `LoggingUtils.setLoggingLevel`, and `LoggingUtils` has only private and package-private methods for converting between the two types. Perhaps the unit tests are not supposed to changed the global log level.